### DISCUSSION TITLE
Ubuntu 18.04 Support

### DISCUSSION THREAD

**Discussion by tdenewiler** [sscpac/statick#451] [General]
End-of-life for Ubuntu 18.04 is rapidly approaching: April 30, 2023.

https://ubuntu.com/about/release-cycle

The Statick developers are not aware of any users that are operating on Ubuntu 18.04. If we do not hear from active users we will begin to phase out support for Ubuntu 18.04. You will still be able to run Statick locally in Ubuntu 18.04 but we will not run automated tests or develop GitHub Actions to use in an Ubuntu 18.04 environment.

**Comment by denewiler** [comment]
Ubuntu 18.04 support was removed from continuous integration in #454.
